`type DOMSelector` is not exported

> This issue proposes a [bug, feature] which...

### Background & Context

With DOMPurify@3.2.0, I got the following error when creating `d.ts` of my project:

``` console
> npx tsc
src/mjs/dompurify.js:5:1 - error TS9006: Declaration emit for this file requires using private name 'DOMPurify' from module '"C:/Users/XXX/Documents/YYY/node_modules/dompurify/dist/purify.es"'. An explicit type annotation may unblock declaration emit.

5 import DOMPurify from 'dompurify';
  ~~~~~~
```

In `dist/purify.es.d.mts`, `type DOMPurify` is not exported.

dist/purify.es.d.mts#L399:
``` typescript
export { type Config, type Hook, type HookName, type RemovedAttribute, type RemovedElement, type UponSanitizeAttributeHook, type UponSanitizeAttributeHookEvent, type UponSanitizeElementHook, type UponSanitizeElementHookEvent, type WindowLike, _default as default };
```

Adding `type DOMSelector` to `export {}` fixed it.

dist/purify.es.d.mts#L399:
``` typescript
export { type Config, type DOMPurify, type Hook, type HookName, type RemovedAttribute, type RemovedElement, type UponSanitizeAttributeHook, type UponSanitizeAttributeHookEvent, type UponSanitizeElementHook, type UponSanitizeElementHookEvent, type WindowLike, _default as default };
```

### Feature

I think this is because DOMPurify is not exported in `src/purify.ts`

src/purify.ts#L1680:
```
interface DOMPurify {
  ...
}
```

Add `export` should fix this.

src/purify.ts#L1680:
```
export interface DOMPurify {
  ...
}
```

I'll send a PR.

--- a/src/purify.ts
+++ b/src/purify.ts
@@ -1677,7 +1677,7 @@ function createDOMPurify(window: WindowLike = getGlobal()): DOMPurify {
 
 export default createDOMPurify();
 
-interface DOMPurify {
+export interface DOMPurify {
   /**
    * Creates a DOMPurify instance using the given window-like object.
    */

